pkg/sfu/downtrack.go

```diff
@@ -467,37 +473,40 @@ func (d *DownTrack) WriteRTP(extPkt *buffer.ExtPacket, layer int32) error {
 	}
 
 	_, err = d.writeStream.WriteRTP(hdr, payload)
-	if err == nil {
-		pktSize := hdr.MarshalSize() + len(payload)
-		for _, f := range d.onPacketSent {
-			f(d, pktSize)
+	if err != nil {
+		d.pktsDropped.Inc()
+		writeErrors := d.writeErrors.Inc()
+		if (writeErrors % 100) == 1 {
```

Comment:
I have seen the repeat error log are all use of closed connection..., it will be better to only suppress that error .

Reply:
Okay, will check on that and make another PR to filter out only that error.